app.nestful.Nestful.metainfo.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>app.nestful.Nestful</id>
+
+  <name>Nestful</name>
+  <summary>The single place for all your lists and tasks</summary>
+
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://nestful.app/terms</project_license>
```

Comment:
is this redistributable? If it is not it needs to use `extra-data`

Reply:
I've converted to `extra-data` but `ar` seems to not be available -- what is the preferred way to have it be available to `apply_extra`?

Reply:
https://github.com/flathub/com.amazon.Workspaces/blob/2f1b8c8bb75c1bf3f24fd0896a3e5844de985fe5/com.amazon.Workspaces.yml#L140-L142

Reply:
You have to pull the desktop file and icon as well here, pick one icon. See https://docs.flatpak.org/en/latest/module-sources.html#extra-data

Reply:
Thanks, I've implemented that. By the time I saw this comment I already added the different-sized icons (I don't have a vector version) -- let me know if you want me to leave only the largest one.